website/source/chapter/cell-eqe-influence.rst

```diff
@@ -0,0 +1,124 @@
+
+.. map-header::
+
+
+Impact of cell-to-cell EQE variation on power output
+====================================================
+
+Introduction
+------------
+Cells in a PV module are typically connected in series, hence module
+performance is limited by the cell generating the lowest current.  Reduced cell
+performance can arise from manufacturing inconsistencies between cells, or
+in-operando events such as localised temperature hotspots and non-uniform
+module degradation.
+
+Previous studies have investigated how external envrionmental factors
+such as non-uniform shading can lead to reduced individual cell or module
+performance resulting in mismatch losses.  While manufacturing inconsistencies
+between cells are often cited as a cause of intramodule mismatch losses, they
+are typically overlooked in the existing body of research.
+
+One way to study the inherent differences between cells is through analyzing
+their external quantum efficiency (EQE).  A cell's EQE curve represents the
+efficiency with which incident photons of different wavelengths generate a
+photocurrent.  Although this spectral effect on PV performance has been studied
+at the module level, no studies have analysed the spectral effect at the cell
+level and its potential role as a driver of mismatch losses at the module
+level.  
+
+To investigate the role of cell-to-cell variation on module performance, the
+following questions are proposed:
+
+#. How does the EQE vary between cells?
+#. What is the impact on PV performance?
+#. Does the performance impact vary seasonally or geographically?
+
+
+Methodology
+----------
+EQE curves for each cell within each of 11 PV modules were measured by the Cell
+Measurement and Performance group at the National Renewable Energy Laboratory.
+These data are in conjunction with simulated meteorological data, including
+spectral irradiance, spanning the contiguous United States at a resolution of
+around 2° longitude/latitude (around 20km) thus covering around 21,000
+individual locations.
+
+The PV devices studied cover a range of technologies, including
+monocrystalline silicon (monoSi) and polycrystalline silicon
+(polySi), and a range of cell architectures including Al-BSF (Aluminium Back
+Surface Field), IBC (Interdigitated Back Contact), PERC (Passivated Emitter
+and Rear Contact), PERT (Passivated Emitter Rear Totally Diffused), and HIT
+(Heterojunction with Intrinsic Thin Layer). IEC characterisation data as well
+as STC and field performance reports of the modules can be found in XXXX.
+XX refer to data section or add references directly here? XX
+
+The spectral mismatch factor, $M$ is used to quantify the change in PV
+performance due to changes in the spectrum.  $M$ represents the ratio of the
+usable fraction (UF) of measured irradiance to the usable fraction of the
+reference irradiance XX IEC reference? XX, and is calculated using the spectral
+response (SR) and spectral irradiance data as follows:
```

Comment:
I think this paragraph and the ones following are getting too far into the weeds.  I think one high-level paragraph is good here, communicating that we're looking at the difference between "worst cell" and "average cell".